packages/communication-react/CHANGELOG.md

```diff
@@ -1,12 +1,32 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Mon, 13 Sep 2021 21:02:16 GMT and should not be manually modified.
+This log was last generated on Tue, 28 Sep 2021 19:19:18 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.0.0-beta.6](https://github.com/azure/communication-ui-library/tree/@internal/react-components_v1.0.0-beta.6)
+
+Tue, 28 Sep 2021 19:19:18 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@internal/react-components_v1.0.0-beta.5..@internal/react-components_v1.0.0-beta.6)
+
+### Changes
+
+* Support editing and deleting sent messages in the `MessageThread` UI
+* Support customization to the `ParticipantList` to inject menu items for each participant
+* (breaking change) API consistency and documentation updates
+  * Add a uniform way to set cutomization options on the composites.
+* Add `MeetingAdapter` to back the `MeetingComposite`. By default, the `MeetingAdapter` is backed by the Azure Communication Services backends.
```

Comment:
@JamesBurnside Do we want to mention `MeetingComposite` in the release notes at this point?

Reply:
Yeah we should be fine too - would think about adding some sort of (BETA) but probably unecessary